### ISSUE TITLE
Duplicate Basler entry on device page

### ISSUE THREAD

**Issue by henrypinkard** [micro-manager/micro-manager.github.io#113]
There are currently both [BaslerCameras](https://micro-manager.org/BaslerCameras) and [BaslerAce](https://micro-manager.org/BaslerAce) entries on the devices page. BaslerAce is the one I wrote. BaslerCameras was added later by someone who worked at Basler named Felix Asche (but before the website was on github). 

So far as I can tell, there is no device adapter corresponding to BaslerCameras. They have this[ application note](https://www.baslerweb.com/fp-1589529716/media/downloads/documents/application_notes/AW00154602000_Getting_Started_with_Micro_Manager_enu.pdf) on their website, which describes how to install MM and setup the HCW, but no instructions about installing a device adapter. Instead, they just use one that is already present, which (I think) is the one I wrote.

In light of this confusion, I propose to delete the BaslerCameras entry

**Comment by marktsuchida** [comment]
I'm also confused....

The only device adapter I see currently is `BaslerPylon`, so we should have a single page with that name (and redirect from both `BaslerCameras` and `BaslerAce`).

The `BaslerPylon` adapter is the code currently in the `DeviceAdapters/Basler` directory. It is also what is used in that application note. This directory [used to be named](https://github.com/micro-manager/mmCoreAndDevices/commit/436adfece7a17631aa203e2941c990ffcad3bbb4) `BaslerAce`, but it looks like `BaslerAce` and `BaslerPylon5` ([by Ravi](https://github.com/micro-manager/micro-manager/commit/1a1e0cf437d8fd415682f1bd7e8001a1c4be19d3), but maybe based on your code?) coexisted in that directory for a time until `BaslerAce` was [removed](https://github.com/micro-manager/mmCoreAndDevices/commit/df1b018abe6e77e7aff992c75a6ca6a2eede232f). `BaslerPylon5` was later renamed to `BaslerPylon6` and then to `BaslerPylon`.

**Comment by henrypinkard** [comment]
I'm pretty sure Ravi copied and pasted my code and made an updated version of it, and then there were two versions until the original that I made disappeared. So there was never really a reason to have two different ones. In any case, I think it makes sense to delete the two current pages and make one `BaslerPylon` that reflects the true situation or merge them into one

**Comment by marktsuchida** [comment]
Yeah, that sounds good to me. You can add redirects by using the following page header (in case you're not already aware):
```yaml
---
title: BaslerPylon
redirect_from:
  - /BaslerAce
  - /BaslerCameras
  - /wiki/BaslerAce
  - /wiki/BaslerCameras
layout: page
---
```
(The ones with 'wiki' redirect from the URLs prior to our move to GitHub Pages.)

**Comment by henrypinkard** [comment]
#168
